CHANGELOG.md

```diff
@@ -73,7 +73,7 @@ The other `type` you can use is `JAEGER` now. The `config` keys and values are J
 
 - [#1284](https://github.com/thanos-io/thanos/pull/1284) Add support for multiple label-sets in Info gRPC service.
 This deprecates the single `Labels` slice of the `InfoResponse`, in a future release backward compatible handling for the single set of Labels will be removed. Upgrading to v0.6.0 or higher is advised.
-*breaking* If you run have duplicate queries in your Querier configuration with hierarchical federation of multiple Queries this PR makes Thanos Querier to detect this case and block all duplicates. Refer to 0.6.1 which at least allows for single replica to work.
+*breaking* If you run have duplicate queries in your Querier configuration with hierarchical federation of multiple Queries this PR makes Thanos Querier to detect this case and block all duplicates. Refer to 0.6.2 which at least allows for single replica to work.
```

Comment:
This is quite vague info TBH. And yes, I wrote this initially ;p 
Let's maybe rephrase it:

```suggestion
*breaking* If you have duplicated `storeAPIs` in your Querier configuration with the hierarchical federation of the multiple Queries this PR makes Thanos Querier to detect this case and block all duplicates. Refer to `master` or future `v0.7.0` builds which allows for a single replica to work in this case.
```

.. as it's not planned to fetch the fix to 0.6.x. If that's necessary let us know. If it would be it should be done while ago. v0.7.0 first rc release is planned as described here:  https://thanos.io/release-process.md/

Reply:
Thanks, pulled in the recommendation.